### PR TITLE
chore(eth-pm): bump @waku/core to 0.0.8 and other @waku/* deps

### PR DESCRIPTION
**Pull request by fryorcraken**

### CODE REVIEW

File: eth-pm/src/App.tsx [hunk 1]
```diff
@@ -67,13 +68,13 @@ const useStyles = makeStyles({
...
 function App() {
-  const [waku, setWaku] = useState<WakuPrivacy>();
+  const [waku, setWaku] = useState<RelayNode>();
   const [provider, setProvider] = useState<Web3Provider>();
   const [encryptionKeyPair, setEncryptionKeyPair] = useState<
     KeyPair | undefined
   >();
   const [privateMessageDecoder, setPrivateMessageDecoder] =
-    useState<AsymDecoder>();
+    useState<IDecoder<DecodedMessage>>();
```
Comment (fryorcraken) [inline eth-pm/src/App.tsx:77]: This is not good:
- Complex type
- Does not provide access to the full API of the `Decoder` (e.g. access to signature).

Need to expose the type in js-waku to fix this (wip).

File: eth-pm/src/App.tsx [hunk 2]
```diff
@@ -2,8 +2,9 @@ import "@ethersproject/shims";
 
 import React, { useEffect, useState } from "react";
 import "./App.css";
-import type { WakuPrivacy } from "@waku/interfaces";
-import { AsymDecoder, SymDecoder } from "@waku/message-encryption";
+import type { RelayNode, IDecoder } from "@waku/interfaces";
+import { createDecoder as createSymmetricDecoder } from "@waku/message-encryption/symmetric";
```
Comment (fryorcraken) [inline eth-pm/src/App.tsx:6]: This is not ideal but I believe it is more of a code smell in the example as decoders for various types of messages are instantiated in the same file.

A better architecture would separate concerns between the various functionalities.

The main issue I believe is the access to the Waku node instance.
Once cleaned up, then this instance can be passed to various component and each component can:
- setup their encoder/decoder
- setup the right hooks on the waku instance (e.g. addObserver)

File: eth-pm/package.json [hunk 3]
```diff
@@ -9,10 +9,10 @@
...
     "@material-ui/icons": "^4.11.3",
     "@waku/byte-utils": "0.0.2",
-    "@waku/core": "0.0.6",
-    "@waku/create": "0.0.4",
-    "@waku/interfaces": "0.0.5",
-    "@waku/message-encryption": "0.0.4",
+    "@waku/core": "^0.0.8",
+    "@waku/create": "^0.0.6",
+    "@waku/interfaces": "^0.0.6",
+    "@waku/message-encryption": "^0.0.7",
```
Comment (danisharora099) [inline eth-pm/package.json:15]: PR title mentions only bumping up `core` -- should we update the title?

**Review by danisharora099 — APPROVED** [review]
lgtm!

**Pull request merged**